backend/api/v1/routes/dashboard.py

```
@@ -0,0 +1,204 @@
+    RecommendationListResponse,
+    RecommendationResponse
+)
+from backend.repositories.monitoring_repository import get_repository
+from backend.models.monitoring import MetricStatus
+
+router = APIRouter(prefix="/api/dashboard", tags=["dashboard"])
+repository = get_repository()
+
+
+@router.get("/summary", response_model=DashboardSummaryResponse)
+async def get_summary(x_tenant_id: str = Header(..., description="고객사 ID")):
```

Ruff (0.13.1)

Unused function argument: `x_tenant_id`

(ARG001)